css/style.css

```diff
@@ -33,51 +32,207 @@ Developer @GA in NYC
 
 .alignright {
   float: right;
-  padding: 0 0 10px 10px; /* note the padding around a right floated image */
+  padding: 0 0 10px 10px; /* note the padding around a right floated image 
 }
 
 .alignleft {
   float: left;
-  padding: 0 10px 10px 0; /* note the padding around a left floated image */
+  padding: 0 10px 10px 0; /* note the padding around a left floated image
 }
 
+*/
+
 /******************************************
 /* BASE STYLES
-/*******************************************/
+/******************************************/
 
 body {
-     color: #000;
-     font-size: 12px;
+     background: #f0efef;
+     color: #fff;
+     font-size: 10px;
      line-height: 1.4;
-     font-family: Helvetica, Arial, sans-serif;
-     background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
-      background-size: cover;
+     font-family: 'Open Sans', sans-serif;
+     background-size: cover;
+     text-align: center;
 }
 
-
 /******************************************
-/* LAYOUT
-/*******************************************/
+LAYOUT
+/******************************************/
 
 /* Center the container */
 .container {
```

Comment:
1. Yikes, yes it is and ID. 
2. I switched this to fixed width.
3. Now it works after the switch!

Thanks!!! - K